packages/styleguide/src/components/ExpandableLink/ExpandableLink.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useRef, CSSProperties, ReactElement, useState } from 'react'
+import ReactDOM from 'react-dom'
+import { useColorContext } from '../Colors/ColorContext'
+import { css } from 'glamor'
+import ExpandableLinkCallout from './ExpandableLinkCallout'
+
+type Props = {
+  children?: React.ReactNode
+  attributes: object
+  title: string
+  description: string
+  href: string
+}
+
+export type StateProps = {
+  href: string
+  title: string
+  description: string
+  position: CSSProperties
+}
+
+const styles = {
+  link: css({
+    cursor: 'pointer',
+    padding: '0 2px',
+    textDecorationLine: 'underline',
+    textDecorationStyle: 'dotted',
+    textDecorationSkip: 'ink',
+    textDecorationThickness: 2,
+    textUnderlineOffset: 3,
+  }),
+}
```

Comment:
the inconsistency triggers me :D one time it's above and one time below the component in this PR

Reply:
I am blind. In what file is style below the component?

Reply:
In `ExpandableLinkCallout` it's in the middle – below `ExpandableLinkP` (which uses one styles rule) and above `ExpandableLinkCallout` (which uses the rest). Generally it more logical to have it above since you should declare variables before using them.